spec/spec_helper.rb

```diff
@@ -57,9 +57,9 @@
   # triggering implicit auto-inclusion in groups with matching metadata.
   config.shared_context_metadata_behavior = :apply_to_host_groups
 
-# The settings below are suggested to provide a good initial experience
-# with RSpec, but feel free to customize to your heart's content.
-=begin
+  # The settings below are suggested to provide a good initial experience
```

Comment:
Happy for this to be amended but would prefer a separate dedicated PR

Reply:
no worries. will push one